CODEOWNERS

```diff
@@ -2,9 +2,23 @@
 # They are code owners by default for the whole repo.
 * @prometheus/default-maintainers
 
-# Example adding a dedicated maintainer for AWS SD, and also "default
-# maintainers" so that they do not need to bypass codeowners check to merge
-# something.
-# Example comes from
+# Subsystems
+/Makefile @simonpasquier @SuperQ @prometheus/default-maintainers
+/cmd/promtool @dgl @prometheus/default-maintainers
```

Comment:
Just double checking, do we need `@prometheus/default-maintainers` if we have `* @prometheus/default-maintainers` in the head of this file?